hack/chains/release-pipeline-with-ec-demo.sh

```diff
@@ -114,6 +119,21 @@ spec:
 
 oc get pipeline simple-release -o yaml | yq e '.spec' -
 
+title "Enterprise Contract Policy"
+
+echo -n '
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ec-policy
+data:
+  # HINT: Add "test" to the list to ignore tests
+  policy.json: |
+      {"non_blocking_checks":["not_useful"]}
+' | oc apply -f - > /dev/null
```

Comment:
I'm going to change this so it only creates the configmap if it's missing to avoid tramping over previously configured policies.

Reply:
Done